# Generated by Django 4.0.6 on 2022-07-08 07:25

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('attendance_tools', '0005_rename_emp_name_attendance_mail_table_username_and_more'),
    ]

    operations = [
        migrations.RenameField(
            model_name='user',
            old_name='phone_no',
            new_name='mobile_no',
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
